src/instructions/register.rs

```diff
@@ -94,6 +94,18 @@ pub trait Register:
     fn from_u32(v: u32) -> Self;
     fn from_u64(v: u64) -> Self;
     fn from_usize(v: usize) -> Self;
+
+    fn ne(self, rhs: Self) -> Self {
+        self.eq(rhs).logical_not()
+    }
+
+    fn greater_or_equal_than(self, other: Self) -> Self {
+        self.less_than(other).logical_not()
+    }
+
+    fn greater_or_equal_than_signed(self, other: Self) -> Self {
+        self.less_than_signed(other).logical_not()
+    }
```

Comment:
How about:
`eq`, `ne`, `gt`, `ge`, `lt`, `le`
https://doc.rust-lang.org/std/cmp/trait.PartialOrd.html

Reply:
Is it weird that `le` is in abbreviation, but `signed` is in full form?

Reply:
Yes, but `eq` vs `less_than` is also wired

Reply:
Which actually brings my second question: should we go with `eq`, `lt`, `lt_signed`, or `equal`, `less_than`, `less_than_signed`?

Reply:
`le_s`, `ge_s`, not bad

Reply:
`greater_or_equal_than` is weird. Usually, we omit the first preposition and get `greater_or_equal_to`, or use the full form `greater_than_or_equal_to`.

Another reason to use abbreviations.

Reply:
Many languages use the abbreviations, such as Python, Lua